src/org/zaproxy/zap/extension/help/ExtensionHelp.java

```diff
@@ -314,6 +321,32 @@ private ZapMenuItem getMenuHelpZapUserGuide() {
 		return menuHelpZap;
 	}
 	
+	private ZapMenuItem getMenuHelpZapHome() {
+		if (menuHelpZapHome == null) {
+			final Object[] options = {"Open", "Ok"};
```

Comment:
- This was meant to be replaced with the resource messages?
 - The variable could be declared in the `actionPerformed` method.